pages/dkp/konvoy/1.8/install/install-onprem/index.md

```diff
@@ -527,6 +522,19 @@ Note that if your stateful workload is using a [local persistent volume][local_p
 If the node fails, the stateful workload might lose its data.
 If you cannot tolerate this limitation, you should consider other storage options.
 
+# Configure NTP on nodes
+
+By default, Konvoy will install and configure `chrony` as an NTP client used to synchronize time on the control-plane and worker machines.
+You can disable this by configuring your cluster as follows:
```

Comment:
Let's make it clear you should only disable this if you already have something else doing this:

```suggestion
If your machines already have time synchronization software installed and configured, you can disable the installation of `chrony` by configuring your cluster as follows:
```